README.md

```diff
@@ -150,3 +148,13 @@ $ keytool -list -v -keystore "$HOME/.android/debug.keystore" \
     https://support.google.com/firebase/answer/7015592
 
 This includes the API key and URL for your new Firebase project.
+
+### Building with Google Cloud Build (optional)
+
+1. Install google-cloud-sdk
+
+2. gcloud init
+ 
+3. gcloud auth login
+  
+4. gcloud config set project [PROJECT_ID]
```

Comment:
Is there a step missing here? What do you do after setting the project to kick off the build?

Reply:
This is only needed if we need to update the docker images.
So the next step would be to build and upload the docker image. 
(mentioned in the file docs of cloud-build/cloudbuild.yaml)
`gcloud builds submit --config=cloudbuild.yaml --substitutions=_ANDROID_VERSION=28`

Reply:
So should the title be updated to "Set up..." rather than "Building..."?

Reply:
That sounds a lot better

Reply:
Maybe "Setup docker images for Google Cloud Build"?

Reply:
Isn't it the other way around? Updated text a bit; let's revise it more in a separate PR as needed. Thanks!